client/lib/explat/assignment.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { stringify } from 'qs';
+
+/**
+ * Internal dependencies
+ */
+import { logError } from './error';
+
+const EXPLAT_VERSION = '0.1.0';
+
+export const fetchExperimentAssignment = async ( {
+	experimentName,
+	anonId,
+}: {
+	experimentName: string;
+	anonId: string | null;
+} ): Promise< unknown > => {
+	// Bail if using IE 11.
+	if ( ! window.fetch ) {
+		return;
+	}
+
+	const params = stringify( {
+		experiment_name: experimentName,
+		anon_id: anonId ?? undefined,
+	} );
+
+	try {
+		const response = await window.fetch(
+			`https://public-api.wordpress.com/wpcom/v2/experiments/${ EXPLAT_VERSION }/assignments/woocommerce?${ params }`
```

Comment:
It seems on Calypso Green we may need to send an OAuth token here for authenticated users (Calypso Blue uses cookies), this is something we have run into for Jetpack.

Reply:
Jetpack connected users will have an actual id, not an anonId which is what we should be sending, if I understand correctly. 

As far as the OAuth token, this code won't be running on Calypso so it won't need to be included. Or will it?